CMakeLists.txt

```diff
@@ -17,7 +17,7 @@ set(GENERATED_INCLUDE_DIR ${CMAKE_CURRENT_BINARY_DIR}/src/inc)
 
 function(clr_unknown_arch)
     if (WIN32)
-        message(FATAL_ERROR "Only AMD64 and I386 are supported")
+        message(FATAL_ERROR "Only AMD64, ARM64 and I386 are supported")
```

Comment:
It's supported in the sense that it builds successfully for x86 (or at least used to when I wrote that if).